optics-th/tests/Optics/TH/Tests/DuplicateRecordFields.hs

```diff
@@ -0,0 +1,47 @@
+{-# LANGUAGE DataKinds #-}
+{-# LANGUAGE DuplicateRecordFields #-}
+{-# LANGUAGE FlexibleInstances #-}
+{-# LANGUAGE MultiParamTypeClasses #-}
+{-# LANGUAGE OverloadedLabels #-}
+{-# LANGUAGE TemplateHaskell #-}
+{-# LANGUAGE TypeFamilies #-}
+{-# LANGUAGE TypeOperators #-}
+{-# LANGUAGE UndecidableInstances #-}
+{-# OPTIONS_GHC -fforce-recomp #-}
```

Comment:
Is this needed?

Reply:
Oops, I left that in by mistake. For some reason changing the TH code wasn't leading to recompilation when I was testing. Fixed.